src/js/angular/core/services/cookie.service.js

```diff
@@ -0,0 +1,76 @@
+const COOKIE_NAME = '_wb=';
+const COOKIE_VERSION = 'WB1';
+const PATH = "; path=/";
+const EXPIRATION = "; expires=Fri, 31 Dec 2099 23:59:59 GMT";
+const EXPIRED = "; Max-Age=-99999999";
```

Comment:
Write documentation for these constants above because they are specific and related to this concrete cookie.

Reply:
done